### PR TITLE
fixes for bsod when attaching to Texas Instruments SmartRF04EB

### PR DESCRIPTION
**Pull request by mdilic**
I ran into BSOD when attaching to Texas Instruments SmartRF04EB. Turns out the fix was easy. Currently works just fine.

### CODE REVIEW

File: driver/vhci/vhci_write.c [hunk 1]
```diff
@@ -71,6 +71,12 @@ post_select_config(pusbip_vpdo_dev_t vpdo, PURB urb)
 	USHORT	len;
 	struct _URB_SELECT_CONFIGURATION	*urb_selc = &urb->UrbSelectConfiguration;
 
+	// If ConfigurationDescriptor is NULL, the device will be set to an unconfigured state.
+	if (urb_selc == NULL || urb_selc->ConfigurationDescriptor == NULL) {
```
Comment (cezuni) [inline driver/vhci/vhci_write.c]: Is it possible that urb_selc is NULL ? If that case will not happen, how about dropping "urb_selc == NULL" ?

File: driver/vhci/vhci_read.c [hunk 2]
```diff
@@ -191,11 +191,20 @@ store_urb_class_vendor_partial(pusbip_vpdo_dev_t vpdo, PIRP irp, PURB urb)
 {
 	struct _URB_CONTROL_VENDOR_OR_CLASS_REQUEST	*urb_vc = &urb->UrbControlVendorClassRequest;
 	PVOID	dst;
+	char* buf;
```
Comment (cezuni) [inline driver/vhci/vhci_read.c]: Trivial but coding style.. 😄 
```
char	*buf
```

**Comment by cezuni** [comment]
@mdilic : great thanks for your work! Please, consider my review. And how about merging your two commits into one?

File: driver/vhci/vhci_write.c (hunk 1 above)
Comment (mdilic) [reply driver/vhci/vhci_write.c]: Removing. This was a leftover from my initial investigation of the crash.

File: driver/vhci/vhci_read.c (hunk 2 above)
Comment (mdilic) [reply driver/vhci/vhci_read.c]: Of course :)

**Comment by cezuni** [comment]
@mdilic : For simple version history, I have squashed your commits into a single commit and merged it into master branch. Thanks for your work!

**Comment by mdilic** [comment]
tnx!

**Pull request closed without merging**